src/lib/export-utils.ts

```
@@ -0,0 +1,373 @@
+          },
+          {
+            title: "Centralized SPC Schedule",
+            description: "View Schedulefly schedule natively",
+            isPlaceholder: false
+          }
+        ]
+      };
+      
+    case 'directory':
+      // Load actual data from JSON files
+      const contactsModule = await import('@/data/contacts.json');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)